puppet/modules/redmine/templates/postgresql_backup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Are these backups taken offsite?

Reply:
Yes, the puppetmaster has Dirvish set up. I'll update it's config once this is live - see http://projects.theforeman.org/projects/foreman/wiki/Redmine#Moving-gears point 9

Reply:
Sone, and confirmed `/usr/share/redmine_data/` is now being backed up as usual.